htlcswitch/packet.go

```diff
@@ -95,9 +95,13 @@ type htlcPacket struct {
 	outgoingTimeout uint32
 
 	// customRecords are user-defined records in the custom type range that
-	// were included in the payload.
+	// were included in the onion payload.
 	customRecords record.CustomSet
 
+	// inWireCustomRecords are custom type range TLVs that are included
+	// in the incoming update_add_htlc wire message.
+	inWireCustomRecords lnwire.CustomRecords
+
```

Comment:
Since we have two of these in the same structure, can we rename the rows to make their differences apparent in the naming?